sandbox/data/cohort.json

```diff
@@ -0,0 +1,681 @@
+                    "patientAgeConstraints": [
+                      {
+                        "minAgeMonths": 6,
+                        "maxAgeMonths": 0
+                      }
+                    ],
+                    "eventDateConstraint": null,
+                    "eventCountConstraint": null,
+                    "preCoordinatedConstraints": [],
+                    "datasets": [
+                      {
+                        "id": "801b8501-9143-458f-9850-2886a2be69f4",
+                        "name": "General Practice Extraction Service (GPES v4)",
+                        "description": "GPES collects information for a wide range of purposes, including providing GP payments. Versions 1, 4 and 5 have been used to directly identify people and apply specific rules to the cohort.",
+                        "acronym": "GPES v4",
+                        "url": "https://digital.nhs.uk/services/general-practice-extraction-service"
+                      },
+                      {
+                        "id": "4e40980e-27f9-4b78-bdcd-7f9c72b9de74",
+                        "name": "GPES Data for Pandemic Planning and Research COVID-19 (GDPPR)",
+                        "description": "The purpose of this data collection is to respond to the intense demand for General Practice data to be shared. This supports the vital planning and research for COVID-19 purposes.",
+                        "acronym": "GDPPR",
+                        "url": "https://digital.nhs.uk/about-nhs-digital/corporate-information-and-documents/directions-and-data-provision-notices/data-provision-notices-dpns/gpes-data-for-pandemic-planning-and-research"
+                      }
+                    ],
+                    "datasetInclusion": []
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "id": "ae0324d0-3fed-4766-bae2-26e089b78b8a",
+        "lastUpdated": "2023-09-25T12:46:17.727Z",
+        "name": "Chronic kidney disease (2022 to 2023)",
+        "nonDigitalPathwayOverview": "If a person receives care from another healthcare provider, for example a hospital, specialist, or community clinic, their GP record is not automatically updated.\nIf a person’s GP record is not up-to-date when creating a cohort, they may not be included in the digital pathway.\nThis may result in a person not receiving the invite for their vaccination.\nIf you believe you’re eligible for a vaccination, but have not received your invite, then [visit the NHS website](https://www.nhs.uk/conditions/vaccinations/) for more information.",
+        "ruleLogicOverview": "The table below shows the clinical rules used to determine who should be contacted or reminded to receive their vaccination based on specific medical conditions.\nThese conditions are detailed in the [Green Book, chapter 19](https://www.gov.uk/government/publications/).",
+        "urlSlug": "22-to-23-chronic-kidney-disease",
```

Comment:
Needs anonymising here too

Reply:
I've taken the year off here, but not in other places it appears. Can you have a look and let me know if that suits please?

Reply:
I think the issue is the kidney disease, since this is a top level condition?

Reply:
(resolved on Slack)